docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    for (i, p) in enumerate(probs):
</pre><p>We don't need <code>enumerate</code>  in both the for loops as we are not using the index <code>i</code>  anywhere?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Outdated, Loschmidt echo removed. (But good catch!)</p>